woocommerce/woocommerce
# fix: CYS - fix path images patterns

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





After the migration to the mono repo, the path of the images is changed. This PR updates the path.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new site either via local environment or JN. Make sure to install WooCommerce Beta Tester as well.
2. Enable `customize-store` feature flag in `Tools -> WCA Test Helper -> Features`
3. Go to `WooCommerce -> Home` and click on `Customize your store` task.
4. Open your browser's inspector and use Chrome's content override [feature](https://developer.chrome.com/docs/devtools/overrides/) to change the content of https://status.openai.com/api/v2/status.json call. Change `status.indicator` to `major`. 
5. Refresh the page, go through the process.
6. Ensure that the patterns have the image



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Fix: CYS - fix path images patterns.

#### Comment 

</details>
